### PR TITLE
Add conduct code

### PR DESCRIPTION
**Pull request by juunegreiros**
Index - desktop
index-desktop

Index - mobile
index-mobile

Conduct Code - desktop
codigo-conduta-desktop

Conduct Code - mobile
codigo-conduta-mobile

**### Changes:** 

- Remove Bower
- Add yarn
- Add conduct code page
- Remove members
- Remove footer

### CODE REVIEW

File: bower_components [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/junegreiros/Documents/sorocabacss.github.io/node_modules/@bower_components
```
Comment (felipesoares6) [inline bower_components]: Do we need this file? :thinking:

File: codigo-de-conduta.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      <svg width="80" height="80" viewBox="0 0 250 250" style="fill:#f9579f; color:#fff; position: fixed; top: 0; border: 0; right: 0;">
+        <path d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"></path>
+        <path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path>
+        <path d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z" fill="currentColor" class="octo-body"></path>
+      </svg>
+    </a>
+    <div class="codigo-de-conduta">
```
Comment (felipesoares6) [inline codigo-de-conduta.html]: code-of-conduct :nail_care:

File: codigo-de-conduta.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      <a href="index.html" class="button-back">
+        <img src="dist/img/left-arrow.svg" alt="Back button">
+      </a>
+      <h1>Código de Conduta</h1>
+      <div class="text">
```
Comment (felipesoares6) [inline codigo-de-conduta.html]: I think we could start to use BEM, what you think about using something like `code-of-conduct__text` here?

File: dist/img/left-arrow.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="512" height="512" viewBox="0 0 492 492"><path fill="#FFF" d="M464.344 207.418l.768.168H135.888l103.496-103.724c5.068-5.064 7.848-11.924 7.848-19.124 0-7.2-2.78-14.012-7.848-19.088L223.28 49.538c-5.064-5.064-11.812-7.864-19.008-7.864-7.2 0-13.952 2.78-19.016 7.844L7.844 226.914C2.76 231.998-.02 238.77 0 245.974c-.02 7.244 2.76 14.02 7.844 19.096l177.412 177.412c5.064 5.06 11.812 7.844 19.016 7.844 7.196 0 13.944-2.788 19.008-7.844l16.104-16.112c5.068-5.056 7.848-11.808 7.848-19.008 0-7.196-2.78-13.592-7.848-18.652L134.72 284.406h329.992c14.828 0 27.288-12.78 27.288-27.6v-22.788c0-14.82-12.828-26.6-27.656-26.6z"/></svg>
```
Comment (felipesoares6) [inline dist/img/left-arrow.svg]: Did you optimized this svg?

File: scss/_codigo-de-conduta.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
+.codigo-de-conduta {
+  h1 {
```
Comment (felipesoares6) [inline scss/_codigo-de-conduta.scss]: Use classes instead of styling direct on tags

File: scss/_codigo-de-conduta.scss [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    margin-top: 5rem;
+    text-align: center;
+
+    @media (min-width: 768px) {
```
Comment (felipesoares6) [inline scss/_codigo-de-conduta.scss]: I think we could add breakpoint variables if we don't have it yet!

**Review by felipesoares6 — CHANGES_REQUESTED** [review]

File: dist/img/left-arrow.svg (hunk 4 above)
Comment (juunegreiros) [reply dist/img/left-arrow.svg]: Yep :P

**Review by felipebernardes — APPROVED** [review]
muito bom! valeu ju

File: scss/common/_variables.scss [hunk 7]
```diff
@@ -5,5 +5,12 @@ $black: #000;
...
 $brilliant-rose: #f857a6;
-$persimmon: #ff5858;
+$persimmon: #ff5858;;
 $pink: #ffc0cb;
+
+//breakpoints
+$md-breakpoint: 768px;
+
+//font-family
+
```
Comment (felipesoares6) [inline scss/common/_variables.scss]: :fire:

File: scss/_conduct-code.scss [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+      width: 70%;
+    }
+
+    &__text {
+      font-weight: 500;
+      line-height: 1.5rem;
+      margin: 1.5rem auto;
+      text-align: left;
+
+      @media (min-width: 768px) {
```
Comment (felipesoares6) [inline scss/_conduct-code.scss]: I think you can use the $md-breakpoint variable here

**Review by felipesoares6 — APPROVED** [review]

**Pull request merged**